src/sleepRepo.js

```diff
@@ -17,13 +17,37 @@ class SleepRepo {
     return Number(avgSleepQuality.toFixed(2))
   }
 
-  getAvgSleepQualityAboveThree() {
-    // need a date 
-      //will you be able to iterate over that date veriable?
-    // need to get 7 days from that date
-    // need to add sleep quality for every user
-    // need to compare that sleep quality avg to 3
-    //
+  getAvgSleepQualityAboveThree(date) {
```

Comment:
@Garrett-Iannuzzi Working through this together and whiteboarding it all out was awesome.  I think this method alone was a huge win from our project.